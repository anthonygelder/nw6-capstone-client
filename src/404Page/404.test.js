import React from 'react';
import ReactDOM from 'react-dom';
import Four from './404';
import renderer from 'react-test-renderer';

describe('Four Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
        < Four />
    , div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Four/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
})