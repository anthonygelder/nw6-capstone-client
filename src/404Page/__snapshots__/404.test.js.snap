// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Four Component renders the UI as expected 1`] = `
Array [
  "   ",
  <br />,
  <br />,
  <h2>
    404 NOT FOUND
  </h2>,
]
`;
